src/app/client/src/app/modules/shared/components/collection-player-metadata/collection-player-metadata.component.html

```diff
@@ -65,4 +65,18 @@
       </div>
     </div>
   </div>
+  <div class="three wide column">
+    <div class="ui list">
+      <div class="item">
+        <div class="content">
+          <div class="header">Owner</div>
+          <div ng-if="metaData.owner" class="description padding-top-1rem">{{metaData.owner}}
+            <a (click)="showContentCredits()">Content Credits</a>
```

Comment:
Cross check all html templates all the text should be coming from resource bundles